src/dialogs/settingsdialog.cpp

```diff
@@ -976,6 +980,23 @@ void SettingsDialog::readSettings() {
     // set the cursor width spinbox value
     ui->cursorWidthSpinBox->setValue(
             settings.value("cursorWidth", 1).toInt());
+
+    //Adding options to the search engine selection combo box
```

Comment:
can you please try to use the coding style of the rest of the code for comments, variable declaration and so on